### DISCUSSION TITLE
Help understanding strata option for anosim/adonis

### DISCUSSION THREAD

**Discussion by mradz19** [vegandevs/vegan#554] [Q&A]
I want to look for differences between my samples based on whether they responded to a drug treatment or not, but I am comparing pre and post treatment samples.

For example, my samples are as follows:

Patient 1 has a pre and post sample and did respond to treatment
Patient 2 has a pre and post sample and did not respond to treatment

I want to know if there are any differences within the microbial community of patients who responded compared to those who didnt

Is this code the correct way to do this?

`adonis(bray_dist ~ sample_data(physeq)$Response, method = "bray", strata=sample_data(physeq)$Treatment)`

Where response is the column of my metadata specifying if they responded or not and treatment is the column specifying if the sample is pre or post treatment

**Comment by gavinsimpson** [comment]
I think you would be better off with `dbrda()` if you must analyse the dissimilarities directly (as opposed to say modelling suitably transformed ASV or OTU data directly). As you have repeated observations on patients, you may need to adjust the permutation design to restrict permutations to be within patients, and that may or may not require you to also use `+ Condition(Patient)` in the formula to remove (partial out) the differences *between* patients, which isn't possible aith `adonis()`.

(And you shouldn't be using `adonis()` anymore, if you must do PERMANOVA use `adonis2()`.)

**Reply by Ivy-ops** [reply]
Hi @gavinsimpson @mradz19, have this issue been solved? 
I have the same question as @mradz19, Is adonis2(bray_dist ~ Response[post vs pre], strata=SubjectID) correct for pre- and post-treatment design analysis? 
Thank you!

**Comment by mradz19** [comment]
Hi Gavin thanks for your help I'm pretty new to this. I have a bray curtis dissimilarity matrix that I am using as input, would you be able to give me an idea of how I would use dbrda()? As in how to write out the script?

As I said I'm pretty new to this so I'm not entirely sure what statistical test is best for the questions I'm trying to answer.
